packages/shared/src/components/sources/SourceCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import SquadMemberShortList from '../squads/SquadMemberShortList';
+import { Card } from '../cards/Card';
+import { Button } from '../buttons/Button';
+import { Connection } from '../../graphql/common';
+import { SourceMember } from '../../graphql/sources';
+
+interface SourceCardProps {
+  title: string;
+  subtitle?: string;
+  icon?: ReactNode;
+  image?: string;
+  description?: string;
+  action?: {
+    text: string;
+    onClick: () => void;
+  };
+  members?: Connection<SourceMember>;
+  membersCount?: number;
+  clampDescription?: boolean;
+}
+
+export const SourceCard = ({
+  title,
+  subtitle,
+  icon,
+  image,
+  description,
+  action,
+  members,
+  membersCount,
+  clampDescription = true,
+  ...props
+}: SourceCardProps): ReactElement => {
+  const items =
+    membersCount > 0 &&
+    members?.edges?.reduce((acc, current) => {
+      acc.push(current.node);
+      return acc;
+    }, []);
+
+  return (
+    <Card className="overflow-hidden p-0">
+      <div className="px-4 h-24 rounded-t-2xl bg-theme-bg-onion" />
+      <div className="flex flex-col flex-1 p-4 -mt-12 rounded-t-2xl bg-theme-bg-secondary">
+        <div className="flex justify-between items-end mb-3">
+          {!!image && (
+            <img
+              className="-mt-14 w-24 h-24 rounded-full z-10"
+              src={image}
+              alt={`${title} source`}
+            />
+          )}
+          {!image && (
+            <div className="flex justify-center items-center -mt-14 w-24 h-24 bg-pepper-40 rounded-full z-10">
```

Comment:
Not quite sure pepper is the right one here.
We might need to introduce a new one instead to accomodate light mode.

Reply:
I have asked the question to design